src/Microsoft.HttpRepl/Preferences/WellKnownPreference.cs

```diff
@@ -3,6 +3,7 @@
 
 using System.Collections.Generic;
 using System.Reflection;
+using System.Security;
```

Comment:
why is this needed?

Reply:
Its not. Nor do I know why it was there. But its gone now.